src/libcore/clone.rs

```diff
@@ -75,6 +75,15 @@ pub trait Clone : Sized {
     }
 }
 
+// FIXME(aburka): this method is used solely by #[deriving] to
+// assert that every component of a type implements Clone.
+//
+// This should never be called by user code.
+#[doc(hidden)]
+#[inline(always)]
+#[stable(feature = "rust1", since = "1.0.0")]
+pub fn assert_receiver_is_clone<T: Clone + ?Sized>(_: &T) {}
```

Comment:
should this be stable or unstable? It needs a new feature name and updated version.